crates/nu-command/src/strings/format/command.rs

```diff
@@ -182,7 +190,9 @@ fn format_record(
                         val: col_name.clone(),
                         span,
                     }]) {
-                    Ok(value_at_column) => output.push_str(value_at_column.as_string()?.as_str()),
+                    Ok(value_at_column) => {
+                        output.push_str(value_at_column.into_string(", ", config).as_str())
```

Comment:
This line is the most important part; the rest is mostly noise from `cargo fmt` and passing `config` through.

Reply:
it took me a minute to understand what was going on here. it's that format_filesize() doesn't exist in as_string and it also requires the config and the `", "` does nothing for strings. it does look like some other types could be added to as_string() because they don't require config but we can save that for another pr.